resources/leiningen/new/xiana/src/backend/app/interceptors.clj

```diff
@@ -0,0 +1,14 @@
+(ns interceptors
```

Comment:
This file shouldn't be in interceptors folder?

Reply:
It follows the structure proposition https://chat.flexiana.com/#narrow/stream/89-open-source/topic/Directory.20structure

Thinking of that, the intent I see there is just to have var imports in this file and to have our own interceptor implementations in the interceptors/ folder. Is this right, @jiriknesl ?

If yes, then I will move the sample to some file in the interceptors/ folder

Reply:
Ok, I've got it.